core/wiki/macros/sidebar-resizer.tid

```diff
@@ -0,0 +1,30 @@
+title: $:/core/macros/sidebar-resizer
+tags: $:/tags/Global
```

Comment:
Why does this need to be a global? Is it used outside of the sidebar-resizer tiddler?

Reply:
Hi @saqimtiaz , I didn't want to put this into the sidebar PageTemplate - that's why I made it a global

Reply:
I recommend importing the procedures only where needed. We should avoid global variables where not strictly necessary.

Reply:
I've done so